src/CoreGraphics/CGColor.cs

```diff
@@ -330,6 +330,20 @@ static public CGColor CreateCmyk (nfloat cyan, nfloat magenta, nfloat yellow, nf
 			var h = CGColorCreateGenericCMYK (cyan, magenta, yellow, black, alpha);
 			return h == IntPtr.Zero ? null : new CGColor (h, owns: true);
 		}
+
+		[iOS (14,0)][TV (14,0)][Watch (7,0)][Mac (10,16)]
+		[DllImport(Constants.AccessibilityLibrary)]
+		static extern /* NSString */ IntPtr AXNameFromColor (/* CGColorRef */ IntPtr color);
+
+		[iOS (14,0)][TV (14,0)][Watch (7,0)][Mac (10,16)]
+		public NSString AXName {
+			get {
+				var h = AXNameFromColor (handle);
+				return Runtime.GetNSObject<NSString> (h);
```

Comment:
if they are not constants then it would be easier to use as `string` and then you can use 
```csharp
return NSString.FromHandle (AXNameFromColor (handle));
```

Reply:
They are not constants since they are locallized, will make the change.